libs/markets/src/lib/components/market-info/info-key-value-table.tsx

```diff
@@ -63,10 +78,35 @@ const Row = ({
       dtClassName={className}
       ddClassName={className}
     >
-      <Tooltip description={tooltipMapping[field]} align="start">
-        <div tabIndex={-1}>{startCase(t(field))}</div>
-      </Tooltip>
-      <span style={{ wordBreak: 'break-word' }}>{formattedValue}</span>
+      <div className="flex items-center gap-3">
+        <Tooltip description={tooltipMapping[field]} align="start">
+          <div tabIndex={-1}>{startCase(t(field))}</div>
+        </Tooltip>
+
+        {valueDiffersFromParentMarket && (
+          <Lozenge className="py-0" variant={Intent.Primary}>
+            {t('Updated')}
+          </Lozenge>
+        )}
+
+        {newValueInSuccessorMarket && (
+          <Lozenge className="py-0" variant={Intent.Primary}>
+            {t('Added')}
+          </Lozenge>
+        )}
+      </div>
+      <div style={{ wordBreak: 'break-word' }}>
+        {valueDiffersFromParentMarket ? (
```

Comment:
suggestion: I think we could use red/green diff pattern

Reply:
thanks for the suggestion - we wanted to avoid the success/failure colouring as it's really just about informing the user of changes occuring